### DISCUSSION TITLE
Two forms on same site-one validates, one does not

### DISCUSSION THREAD

**Discussion by chiester** [vazco/uniforms#1059] [Q&A] [answered]
I have two forms that are set up almost identically. There are differences in their schemas, fields, etc.

One of them, when I submit it, the validation errors fields show up. 

For the other, even if required fields are missing, etc. the error fields do not render. I can see in the bridge that the validation fails and there are error messages, but the form does not show the error fields. 

How can I track this down? 

Is there some limitation, like two different React components with Uniforms can't have a schema called `schema` or something like that? 

Also, even though the validation fails, the form is still submitted.

**Comment by chiester** [comment, marked as answer]
I got it to work. I'm not sure what made it start working. Disregard. My apologies for the noise. I would delete this if I could.

**Reply by wadamek65** [reply]
No worries, glad your issue was solved.

**Comment by radekmie** [comment]
Hi @chiester! You've said it's already solved, but let me answer one of your questions.

> [...] I can see in the bridge that the validation fails and there are error messages, but the form does not show the error fields. How can I track this down?

Make sure, that there's a validation error in your form - that's already half of the success. Once you've checked it, there are two possibilities: either the bridge is not able to extract the information from the error object (due to a bug or because the error is not structured correctly), or the field this error is referring to is not rendered.
